src/sagas.js

```diff
@@ -153,6 +162,12 @@ export default function* networkEventsListenerSaga(
   }: Arguments = {},
 ): Generator<*, *, *> {
   yield fork(netInfoChangeSaga, timeout, pingServerUrl, withExtraHeadRequest);
+  yield takeLatest(
```

Comment:
`yield takeLatest(...)` has to be placed at the end of the generator, otherwise it will block forking `connectionIntervalSaga` if a `checkConnectionInterval` is provided.

Once that's fixed, I'll merge it and release a new minor version